pkg/sql/plan2/function/operator/cast.go

```diff
@@ -1010,6 +1106,57 @@ func CastStringAsDecimal128(lv, rv *vector.Vector, proc *process.Process) (*vect
 	return vec, nil
 }
 
+func CastDatetimeAsString(lv, rv *vector.Vector, proc *process.Process) (*vector.Vector, error) {
+	var err error
+	lvs := lv.Col.([]types.Datetime)
+	col := &types.Bytes{
+		Data:    make([]byte, 0, len(lvs)),
+		Offsets: make([]uint32, 0, len(lvs)),
+		Lengths: make([]uint32, 0, len(lvs)),
+	}
+	if col, err = typecast.DateTimeToBytes(lvs, col); err != nil {
+		return nil, err
+	}
+	if err = proc.Mp.Gm.Alloc(int64(cap(col.Data))); err != nil {
```

Comment:
what is this for?

Reply:
I just cal the memory cost in the process